OrrRock/OrrRock/RouteFindingUpload/RouteFindingCameraViewController.swift

```diff
@@ -172,7 +188,23 @@ private extension RouteFindingCameraViewController {
         config.selectionLimit = 1
         let picker = PHPickerViewController(configuration: config)
         picker.delegate = self
-        present(picker, animated: true, completion: nil)
+        
+        PHPhotoLibrary.requestAuthorization({ status in
+            switch status {
+            case .authorized:
+                DispatchQueue.main.async {
+                    self.present(picker, animated: true, completion: nil)
+                }
+            case .limited:
+                DispatchQueue.main.async {
+                    self.present(picker, animated: true, completion: nil)
+                }
```

Comment:
혹시 . authorized 와 .limited에서 같은 행동을 수행하는데 구분하신 이유가 있을까요?
```suggestion
            case .authorized, .limited:
                DispatchQueue.main.async {
                    self.present(picker, animated: true, completion: nil)
                }
           
```

Reply:
오, 좋은 것 같아요! 바로 반영하겠습니다. 감사합니다 도도 :)